indico/htdocs/sass/modules/_event_display.scss

```diff
@@ -20,6 +20,8 @@
 @import 'partials/buttons';
 
 
+$header-blue = #1a64a0;
```

Comment:
Yep. This should be specific to this "theme". Anyway, all the event display definitions for this template are now in a dedicated SASS file.